earth_enterprise/src/fusion/portableglobe/polygontoqtnodes.h

```diff
@@ -80,6 +90,8 @@ class Polygons {
 
   explicit Polygons(const std::string& kml_file);
 
+  explicit Polygons(const kml_as_string&, const std::string& kml_string);
```

Comment:
explicit is only needed on single parameter constructors.

Reply:
Ok.  The PR is updated with this change.